internal/pkg/mapper/defaultbucket/defaultbucket_test.go

```diff
@@ -17,8 +17,8 @@ func createStateWithMapper(t *testing.T) (*state.State, *testdeps.TestContainer)
 	mockedState := d.EmptyState()
 	mockedState.Mapper().AddMapper(defaultbucket.NewMapper(mockedState))
 
-	// Preload the ex-db-mysql component to use as the default bucket source
-	_, err := mockedState.Components().Get(model.ComponentKey{Id: "keboola.ex-db-mysql"})
+	// Preload the keboola.ex-aws-s3 component to use as the default bucket source
```

Comment:
`ex-db-mysql` -> `ex-aws-s3`
MySQL extractor nepouziva default bucket, Aws S3 ano.
Vo funkcionalnych testoch to je spravne, ale v unit to bolo nespravne, tak som to tiez fixol.